src/config/graphical-session/xorg.md

```diff
@@ -82,3 +82,19 @@ Section "InputClass"
   MatchIsKeyboard "on"
 EndSection
 ```
+
+# startx
+
+The [startx(1)](https://man.voidlinux.org/startx.1) script is a frontend to
+[xinit(1)](https://man.voidlinux.org/xinit.1), which can be used to start X
+sessions from the console. For example, to use i3, edit `~/.xinitrc` to contain:
+
+```
+exec /bin/i3
+```
+
+Then call `startx` to start an i3 session:
+
+```
+$ startx
+```
```

Comment:
This feels like duplicated instructions. Perhaps just the "Then call..." line? And in this case, I'd argue that it shouldn't be a man pages link either.

Reply:
Gah, sorry about that - i agree, on both points. :-) i've removed the example command from here, and modified #248.